_pages/en_US/include/seedminer.txt

```diff
@@ -0,0 +1,68 @@
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+To install boot9strap on your device, we derive your device's unique encryption key. To accomplish this, we use a tool called Seedminer to calculate the data encryption key (movable.sed) for your device.
+
+For information on how Seedminer works, see [this presentation](https://zoogie.github.io/web/34⅕c3).
+
+This method uses a powerful graphics card to perform the calculations needed. A volunteer-run website is used for the purpose of assisting you with this method.
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+
+### Section I - Updating the System
+
+In this section, you will update your system to the latest version, which is necessary for Seedminer.
+
+1. Update your device by going to System Settings, then "Other Settings", then going all the way to the right and using "System Update"
+  + The updater may display a message saying "Your system is up to date" instead of updating. This is normal if you are already up to date; continue with the next section
+  + If this gives you an error, set both your DNS settings and Proxy settings to "auto"
```

Comment:
capitalize auto

Reply:
make sure you fix that in finalizing as well, copied that from there